applications/aihorde-application.md

```diff
@@ -0,0 +1,202 @@
+anyone, free and without any strings attached.
+
+The horde initially started as the **KoboldAI Horde**, then forked to the 
+**Stable Horde**. Lately the two have been merged back into a single unified
+**AI Horde**.
+
+## Project Description
+
+The AI Horde has effectively 3 components
+
+* The Worker Inference Backend
+* The AI Horde middleware
+* The Frontends
+
+The part that I developed from scratch is the 
+**[AI Horde middleware](https://github.com/db0/AI-Horde)**.
+It’s a custom-built smart-queuing system built on a stack of
+HAproxy/Python/Postgresql that simply provides a REST API (https://aihorde.net/api)
+that anyone can integrate with without any requirements, even anonymously. 
+
+It is optimized to scale horizontally with multiple cheap VMs in a way a single
+person can handle. Currently it has around 10 VMs for the whole service, all
+configured using Ansible and running on multiple VPS providers.
+
+The AI Horde is crowdsourced, which is where the **Worker Inference Backends** come
+in. These are client software running on people’s PCs which pick up jobs from the AI
+Horde, process them and submit them back. We have multiple for different workloads
+such as Stable Diffusion and LLM, but the core one is the
+[AI Horde Worker](https://github.com/db0/AI-Horde-Worker).
+As I am not a ML developer, I rely on other libraries to perform this task. Currently
+we’re in the process of switching to a new ML inference library. This is the only area where
+[I have other python developers helping me](https://pypi.org/project/hordelib/),
+which is really helping.
+
+Finally, as I mentioned, the AI Horde only provides a REST API. As such, I rely
+on third parties to develop **Frontends and integrations.** While I have developed
+[my own frontend](https://dbzer0.itch.io/lucid-creations/) using the Godot Engine,
+I absolutely do not have enough time to keep it up to date. Fortunately a lot of
+people are seeing the value of the AI Horde and have developed multiple frontends
+to it. Examples include (see https://aihorde.net frontpage for more):
+
+* [Stable UI](https://aqualxx.github.io/stable-ui/dashboard)
+* [ArtBot](https://tinybots.net/artbot)
+* [KoboldAI Lite](https://lite.koboldai.net)
+* [Silly Tavern](https://github.com/Cohee1207/SillyTavern)
+* [Mastodon Bot](https://sigmoid.social/@stablehorde_generator)
+* [Discord Bot](https://github.com/ZeldaFan0225/Stable_Horde_Discord)
+
+All these together are serving thousands of users per day currently.
+
+Also check the [FAQ](https://github.com/db0/AI-Horde/blob/main/FAQ.md).
+
+## Project Scope
+
+I am trying to democratize access to generative AI. I wish to prevent the
+monopoly of this powerful technology by allowing the more optimal sharing
+of resources and therefore the cooperation of human talent.
+
+I did this project as Free/Libre and Open Source so that it can survive any
+challenges, even from myself.  
+
+I also want to create a community of AI Enthusiasts which is explicitly
```

Comment:
Have we given thought to the community spaces we intend to use? I suspect this will just be some public channels in the Nivenly discord, and GitHub to start? Is that a reasonable assumption?